src/data/markdown/docs/40 xk6-disruptor/04 Examples/01 Inject Grpc faults into Service.md

```diff
@@ -0,0 +1,443 @@
+---
+title: 'Inject gRPC faults into Service'
+excerpt: ''
+---
+
+This example shows how [ServiceDisruptor](/javascript-api/xk6-disruptor/api/servicedisruptor) can be used for testing the effect of faults injected in the gRPC requests served by a service.
+
+You will find the complete [source code](#source-code) at the end of this document. Next sections examine the code in detail.
+
+The example uses [grpcpbin](https://grpcb.in), a simple request/response application that offers endpoints for testing different gRPC requests.
+
+The test requires `grpcpbin` to be deployed in a cluster in the namespace `grpcbin` and exposed with an external IP. the IP address is expected in the environment variable `GRPC_HOST`.
+
+You will find the Kubernetes manifests and the instructions of how to deploy it to a cluster in the [test setup](#test-setup) section at the end of this document. You can learn more about how to get the external IP address in the [expose your application](/javascript-api/xk6-disruptor/get-started/expose-your-application) section.
```

Comment:
```suggestion
For the Kubernetes manifests and the instructions on how to deploy it to a cluster, refer to the [test setup](#test-setup) section at the end of this document. To learn more about how to get the external IP address, refer to [Expose your application](/javascript-api/xk6-disruptor/get-started/expose-your-application).
```

Reply:
Just trimming the language a bit, and capitalizing the title to indicate that it's a separate page.